### PR TITLE
Friendly display names

### PR DESCRIPTION
**Pull request by brianpeiris**
- Show new `communityIdentifier` presence field in avatar nametag in pause mode, and in user info panel.
- Fix flow for unverified discord users
- Show avatar thumbnail in user info panel

Goes with https://github.com/mozilla/reticulum/pull/178

### CODE REVIEW

File: src/assets/stylesheets/client-info-dialog.scss [hunk 1]
```diff
@@ -1,11 +1,23 @@
 @import 'shared';
 
+:local(.client-info-dialog) {
+  .dialog__box__contents__title {
```
Comment (gfodor) [inline src/assets/stylesheets/client-info-dialog.scss]: /shrug

File: src/react-components/client-info-dialog.js [hunk 2]
```diff
@@ -61,26 +65,44 @@ export default class ClientInfoDialog extends Component {
...
+  render() {
+    const profile = this.getPresenceProfile();
+    if (!profile) return null;
+
+    const { displayName, communityIdentifier } = profile;
+    const { hubChannel, clientId, onClose } = this.props;
+    const title = (
+      <div className={styles.title}>
+        {displayName}
+        <div className={styles.communityIdentifier}>{communityIdentifier}</div>
```
Comment (gfodor) [inline src/react-components/client-info-dialog.js]: can `communityIdentifier` be null?

**Comment by gfodor** [comment]
lgtm, one minor comment

File: src/assets/stylesheets/client-info-dialog.scss (hunk 1 above)
Comment (brianpeiris) [reply src/assets/stylesheets/client-info-dialog.scss]: Heh, I'm exploiting our old BEM style CSS. I think I've become partial to @netpro2k's CSS structure now -- with `:local()` containers but plain `.foo-bar` sub class names. That would better allow for cases like this where we want to override child component styling. Not sure if there's a better way.

File: src/react-components/client-info-dialog.js (hunk 2 above)
Comment (brianpeiris) [reply src/react-components/client-info-dialog.js]: Yup, it could be null or undefined. But this will just display nothing, as intended.

**Pull request merged**